webhooks/mutatepod_parts.go

```diff
@@ -42,15 +42,15 @@ func k8sSATokenVolume(
 						Path:              K8sSATokenName,
 					},
 				}},
-				DefaultMode: pointer.Int32(288), // 440
+				DefaultMode: pointer.Int32(444),
```

Comment:
This solves the requirement of root user, 288 isn't correct and 444 allows read from everybody